### DISCUSSION TITLE
Implement Structured Reference

### DISCUSSION THREAD

**Discussion by cdbeauchamp7** [ClosedXML/ClosedXML#2123] [General]
Hi,

Is there anything that I can do to request the Structured References to be implemented?  Or anything that I can do to help with this feature?  I just started using this library and of course the first excel sheet that I need to extract from uses structured references.

Thank you!

**Comment by jahav** [comment]
No thank you. Structured references need a different parser which is being [worked on](https://github.com/ClosedXML/ClosedXML.Parser), but features are on back burner.

**Reply by cdbeauchamp7** [reply]
Right on, thank you for the swift reply!
